### ISSUE TITLE
Footer section Accessibility related - Focus order not logical

### ISSUE THREAD

**Issue by AtamDhillon** [govau/design-system-site#750]
This issue was found with SM site as well.

Type: WCAG 2.1 failure
Fails 2.4.3 Focus Order - Level A
Location: Footer
Platform: Desktop
Technology: All
Severity: Low
What’s the issue?
When I navigate on desktop screen using the Tab key, the links under "Community" and such as "More projects", "Accessibility" and "Disclaimer and copyright" does not receive focus in the expected order. Visually "More projects" link is presented to the left and should receive focus first. It works fine as per mobile first design on smaller screens but not on a non-mobile device screen size.

**Comment by rossmullen** [comment]
Thanks @AtamDhillon good find! We've added this to our backlog, [this post on the community Slack channel](https://govau-guides.slack.com/archives/C4QC1D743/p1619568592001500) explains our priority over the next few months.
